docs/rules/no-duplicate-case.md

```diff
@@ -0,0 +1,61 @@
+# Rule to disallow a duplicate case label (no-duplicate-case)
+
+This inspection reports any duplicated case labels on JavaScript switch statements.
```

Comment:
This section needs a lot more information. This should introduce the problem without discussing the rule.